src/components/course/data/service.js

```diff
@@ -59,7 +59,7 @@ export default class CourseService {
     };
   }
 
-  async fetchAllCourseRecommendations(activeCatalogs) {
+  async fetchAllCourseRecommendations(activeCatalogs = []) {
```

Comment:
what was the reason for this change?

Reply:
Just setting a default value for catalogs in case it is not passed explicitly. Like we did in one of the test cases.